docs/contributing.md

```diff
@@ -26,7 +26,7 @@ accept your pull requests.
 ## Building Projects
 
 There are a couple of commands that will help you make working with these
-modules a little easier:
+libraries a little easier:
```

Comment:
+1 This was my only fear.

sw-build and sw-cli will **have** to be downloaded and uses as NPM modules. Everything else is a SW library.